test/e2e/network/loadbalancer.go

```diff
@@ -1585,7 +1648,7 @@ var _ = common.SIGDescribe("LoadBalancers ESIPP [Slow]", func() {
 			if err != nil {
 				return false, nil
 			}
-			if strings.HasPrefix(clientIP, subnetPrefix[0]+".") {
+			if subnetPrefix.Contains(net.ParseIP(clientIP)) {
```

Comment:
if you see the test failure

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/114027/pull-kubernetes-e2e-gci-gce-ingress/1595217512394395648

The error message doesn't show the IP

> Nov 23 01:48:00.570: FAIL: Source IP WAS preserved even after ESIPP turned off. Got , expected a ten-dot cluster ip.

that means that most probably we need to sanitize the output of `GetHTTPContent(i` usually with TrimSpaces, and assert that net.ParseIP() is != nil and log and retry if it is

Reply:
Guess we want retry as the IP might not be ready ?

Reply:
yes,